guidelines: fix Chapter 5

martha-thomae opened the pull request:
In this PR, I am fixing chapter 05 on mensural notation:
- Removing comments from the text
- Minor corrections like spacing and missing words
- Fix examples: removing misleading mensuration signs, adding a comment about the mensuration of the examples (because it is encoded in the `<scoreDef>` section that isn't visible), fixing one example that was completely changed

I also added a few minor things:
- Adding image for plicas
- Adding missing dot section (very short, not a big deal)

martha-thomae committed:
Fix spacing

martha-thomae committed:
Fix examples
Either remove mensuration signs that were not appropriate, add comments to show what the mensuration is in staffDef, fix the implicit-mensuration example (which was changed completely), fix the end tag for verovio in example 'patial-imp-03-remotam.mei'

martha-thomae committed:
Comment out place holder texts

martha-thomae committed:
Add a few words in the introduction section

martha-thomae committed:
Add example for plica

martha-thomae committed:
Add short section for dots

martha-thomae commented:
@musicEnfanthen and @bwbohl would you mind seeing if everything looks well with the image example I added for the plica section?

I can build the website using Oxygen, but I cannot see the examples (neither the ones rendered with Verovio nor those given by images).

Thank you!

martha-thomae committed:
Make comments consistent between examples

martha-thomae commented:
> @musicEnfanthen and @bwbohl would you mind seeing if everything looks well with the image example I added for the plica section?
> 
> I can build the website using Oxygen, but I cannot see the examples (neither the ones rendered with Verovio nor those given by images).
> 
> Thank you!

Never mind, I was able to use the CLI instructions for building the guidelines (using `ant`) and the image seems ok.

martha-thomae commented:
I forgot to say that this is supposed to be for the next release (version 5.0). This is just a fix of things that were wrong in the documentation, extra comments, some things that got changed when moving back to ODD, etc.

bwbohl commented:
> I forgot to say that this is supposed to be for the next release (version 5.0). This is just a fix of things that were wrong in the documentation, extra comments, some things that got changed when moving back to ODD, etc.

Thanks for the hint; it's already in the MEI 5.0 Milestone ;-)

annplaksin reviewed (approved):
These changes look fine to me.
Thanks a lot!

ahankinson commented on source/docs/05-mensural.xml:
```diff
@@ -92,8 +92,8 @@
             <p>This chapter describes the module for encoding mensural notation from the late 13th century to about 1600. Historically, mensural notation preceded the development of Common Music Notation (CMN) and it included a wide range of features that persist in CMN and that can be encoded in a standard manner in MEI. In mensural notation, pitches are notated as in CMN, leaving out here the m ...
-            <p>There are a certain number of differences, however, regarding the representation of duration in mensural notation. The mensural module introduces specific attribute values for notes and rests for appropriately encoding mensural durations. One of the main particularities is that the actual duration of a note is not given only by its symbol but also by position and the context in whi ...
-            <p>Another particularity of mensural notation is the use of proportions that are indicated by numeric proportions or by specific mensuration signs. The proportions indicate that the durations have to be modified accordingly and they can be combined in a very complex manner. Over time, proportions and mensuration signs were simplified and became time signatures in CMN. The attributes a ...
+            <p>There are a certain number of differences, however, regarding the representation of duration in mensural notation. The mensural module introduces specific attribute values for notes and rests for appropriately encoding mensural durations. One of the main particularities is that the actual duration of a note is not given only by its symbol but also by the position and the context in ...
```
```suggestion
            <p>There are a number of differences, however, in the representation of duration in mensural notation. The mensural module introduces specific attribute values for notes and rests for appropriately encoding mensural durations. One of the main differences is that the duration of a note is not determined by its symbol, but also by the position and the context in which the symbol appears in relation to other notes and rests in the same voice. The context is given by one of the 16 mensural <hi rend="italic">species</hi> provided by four levels of division: <hi rend="italic">modus major</hi>, <hi rend="italic">modus minor</hi>, <hi rend="italic">tempus</hi> and <hi rend="italic">prolatio</hi>. Depending on the specific context where the note is positioned, certain rules must be applied in order to determine the duration of a note. In these cases, encoding both the sign and its actual duration is highly recommended.</p>
```

ahankinson replied:
"...encoding both the sign and its actual duration..." I'm not sure what this means? And no examples are provided.

martha-thomae replied:
I understand what you mean by these corrections. Still, I think the original text was referring to the "general context" (the mensuration) as the "context", and it was referring to the "specific context" (notes preceding and following) as the "position" of the note. So, I will accept these corrections and edit the text a bit more so that it becomes clear.

And regarding your comment:
>"...encoding both the sign and its actual duration..." I'm not sure what this means? And no examples are provided.

The example is provided later on in the section about durations. I will try to make that clear.

For both corrections, see commit 6405526

ahankinson commented on source/docs/05-mensural.xml:
```diff
@@ -92,8 +92,8 @@
-            <p>There are a certain number of differences, however, regarding the representation of duration in mensural notation. The mensural module introduces specific attribute values for notes and rests for appropriately encoding mensural durations. One of the main particularities is that the actual duration of a note is not given only by its symbol but also by position and the context in whi ...
-            <p>Another particularity of mensural notation is the use of proportions that are indicated by numeric proportions or by specific mensuration signs. The proportions indicate that the durations have to be modified accordingly and they can be combined in a very complex manner. Over time, proportions and mensuration signs were simplified and became time signatures in CMN. The attributes a ...
+            <p>There are a certain number of differences, however, regarding the representation of duration in mensural notation. The mensural module introduces specific attribute values for notes and rests for appropriately encoding mensural durations. One of the main particularities is that the actual duration of a note is not given only by its symbol but also by the position and the context in ...
+            <p>Another particularity of mensural notation is the use of proportions that are indicated by numeric ratios or by specific mensuration signs. The proportions indicate that the durations have to be modified accordingly and they can be combined in a very complex manner. Over time, proportions and mensuration signs were simplified and became time signatures in CMN. The attributes and el ...
```
```suggestion
            <p>Another difference is the use of proportions that are indicated by numeric ratios or by specific mensuration signs. The proportions indicate that the durations have to be modified and they may be combined. Proportions and mensuration signs were eventually simplified and became time signatures in CMN. The attributes and elements that are necessary for encoding proportions and mensural signs are made available by the module.</p>
```

ahankinson replied:
"The attributes and elements that are necessary for encoding proportions and mensural signs are made available by the module"

This sentence seems redundant. Where else would they be defined?

ahankinson commented on source/docs/05-mensural.xml:
```diff
@@ -397,6 +397,22 @@
                      </specList>
                   </p>
+                  <figure>
+                     <head>Encoding and rendition of plicas</head>
```
```suggestion
                     <head>Encoding and display of plicas</head>
```

ahankinson replied:
"rendition" is not generally a form of "render."

ahankinson commented on source/docs/05-mensural.xml:
```diff
@@ -397,6 +397,22 @@
+               </div>
+               <div xml:id="dots" type="div3">
+                  <head>Dots</head>
+                  <p>Dots of division and augmentation can be encoded by using the <gi scheme="MEI">dot</gi> element. This element is meant to be used as a child of <gi scheme="MEI">layer</gi> following the <gi scheme="MEI">note</gi> or <gi scheme="MEI">rest</gi> after which it appears in the original source.</p> 
```
```suggestion
                  <p>Dots of division and augmentation can be encoded by using the <gi scheme="MEI">dot</gi> element. This element is meant to be used as a child of <gi scheme="MEI">layer</gi> following the <gi scheme="MEI">note</gi> or <gi scheme="MEI">rest</gi> after which it appears in the original source.</p> 
```

My understanding is that dots of division and augmentation alter the duration of notes and rests before, but please correct me if I'm wrong. But I think it's better to not refer to "the original source" and instead focus on describing the effect of the dot on the notes or rest. (Otherwise it just sounds like you're saying "place the dot where the dot should be placed", which isn't very helpful for describing what it does or why it's there.) 

You don't actually explain what the difference is between the different types of dots, or how they function, anywhere in this section.

martha-thomae replied:
Well, the augmentation dot definitely changes the duration of the note preceding it (because this dot adds half of the note's value). However, that is not the case for the dot of division. A dot of division changes the way you would interpret the notes close to it, but not necessarily the note preceding it. For example, if you have `breve, semibreve, semibreve, semibreve, breve` you would keep the first breve perfect and everything else will have a regular value. If you add a dot between the first and second semibreve (i.e., `breve, semibreve, dot, semibreve, semibreve, breve`), then the first breve will be imperfected by the following semibreve, and the last (the third) semibreve will be altered. The dot of division changes the perfect groupings of notes and, therefore, the interpretation of one or more notes in the sequence.

martha-thomae replied:
> But I think it's better to not refer to "the original source" and instead focus on describing the effect of the dot on the notes or rest. (Otherwise it just sounds like you're saying "place the dot where the dot should be placed", which isn't very helpful for describing what it does or why it's there.)

Yes, that is what I am saying, because we usually place the dots of a dotted note (which is basically also an augmentation dot) as a child or attribute of that note element. So I want to make it clear where to place it, just as the sibling following that note. Is there a better way to make this clear?

martha-thomae replied:
> You don't actually explain what the difference is between the different types of dots, or how they function, anywhere in this section.

I am trying to clarify this and point to the examples in the commit [a4d1604](https://github.com/music-encoding/music-encoding/pull/1191/commits/a4d1604f4dbbac17f5649e8037fe2bd4d02e2a51)

ahankinson commented on source/docs/05-mensural.xml:
```diff
@@ -397,6 +397,22 @@
+               <div xml:id="dots" type="div3">
+                  <head>Dots</head>
+                  <p>Dots of division and augmentation can be encoded by using the <gi scheme="MEI">dot</gi> element. This element is meant to be used as a child of <gi scheme="MEI">layer</gi> following the <gi scheme="MEI">note</gi> or <gi scheme="MEI">rest</gi> after which it appears in the original source.</p> 
+                  <p>Dots are not supposed to be encoded as children of notes or rests, but rather as a sibling of these. They are also not supposed to be encoded as attributes (the use of the @dot attribute in a <gi scheme="MEI">note</gi> or <gi scheme="MEI">rest</gi> elements is only acceptable in Common Music Notation, not mensural).</p>
```
```suggestion
                  <p>Dots in mensural notation are not encoded as children of notes or rests, but rather as a sibling of these. They are also not encoded as attributes (the use of the @dot attribute in a <gi scheme="MEI">note</gi> or <gi scheme="MEI">rest</gi> elements is only acceptable in Common Music Notation, not mensural).</p>
```

ahankinson replied:
Changed to be more direct... "supposed" is like giving a suggestion, but I believe it's actually an error, so they absolutely should not be.

martha-thomae committed:
Update the introduction part regarding 'proportions'

Co-authored-by: Andrew Hankinson <andrew.hankinson@gmail.com>

martha-thomae committed:
Fix word

Co-authored-by: Andrew Hankinson <andrew.hankinson@gmail.com>

martha-thomae committed:
Correct 'dots' section

Co-authored-by: Andrew Hankinson <andrew.hankinson@gmail.com>

martha-thomae committed:
Update introduction text about note duration

Co-authored-by: Andrew Hankinson <andrew.hankinson@gmail.com>

martha-thomae committed:
Fix introductory section about duration

martha-thomae committed:
Add details to dot section
Defining types of dots and pointing to examples

martha-thomae commented:
How about now @ahankinson ?

lpugin commented on source/docs/05-mensural.xml:
```diff
@@ -398,7 +398,7 @@
                   </p>
                   <figure>
-                     <head>Encoding and rendition of plicas</head>
+                     <head>Encoding and display of plicas</head>
```
```suggestion
                     <head>Encoding and rendering of plicas</head>
```

martha-thomae committed:
Fix word (rendering)

Co-authored-by: Laurent Pugin <lxpugin@gmail.com>

bwbohl commented:
@ahankinson as you requested changes, could you please have a second look at whether everything is fine now?

martha-thomae commented:
I think most (if not all) of the points were addressed. And, if June 15th is a hard deadline, I can still make fixes in future minor releases as well.

musicEnfanthen commented:
Thank you @martha-thomae . So this PR would be ready for final review?

martha-thomae commented:
Yes, I think so.

martha-thomae committed:
Add older sections that were removed
These sections might be empty, but at least they give an idea of other things available in MEI and general information about them, to be filled out later by others. It helps for others to contribute these sections to the guidelines

martha-thomae committed:
Similarly add placeholders for examples for others to fill in

martha-thomae committed:
Fix comment

martha-thomae committed:
Clarification of the provenance of the 'dot' element
The other three following sections are also part of MEI.shared module

martha-thomae committed:
Fix typo

martha-thomae committed:
Merge pull request #1 from martha-thomae/mensuralchap_more

Add back comments lost since v3

Add back commented sections indicating future work. As Anna Plaksin (Mensural IG co-chair) indicated, this information was useful for people to know how accidentals, custos, and coloration work in general or that they are available somewhere in the schema (even if full details are not provided yet). These sections have to be filled out later, but the fact that they exist helps people to have an idea of what is available in the schema and also helps us to allow others to contribute to the guidelines by filling in these sections.

I also used the same style for commenting on places where examples are expected to be provided by the musicologists of the community.

martha-thomae commented:
Sorry, I couldn't resist. I made a final set of commits (via an internal pull request) to add back a few commented sections which were really useful and that were erased somewhere along the way. I know @annplaksin will back me up on this.

If you don't want me to add this back for this release because we are introducing too many changes, I can revert this last commit. But I think they are relatively minor changes and these sections (although still waiting to be properly filled in) were useful (as indicated in fe3b026).

martha-thomae committed:
Add reference to mensuration and proportions sections

musicEnfanthen commented on source/docs/05-mensural.xml:
```diff
@@ -397,6 +397,35 @@
+               <div xml:id="dots" type="div3">
+                  <head>Dots</head>
+                  <p>Dots of division and augmentation can be encoded by using the <gi scheme="MEI">dot</gi> element (provided by the MEI.shared module). This element is meant to be used as a child of <gi scheme="MEI">layer</gi> following the <gi scheme="MEI">note</gi> or <gi scheme="MEI">rest</gi> after which it appears in the original source.</p> 
+                  <p>Dots in mensural notation are not encoded as children of notes or rests, but rather as a sibling of these. They are also not encoded as attributes (the use of the <att>dot</att> attribute in a <gi scheme="MEI">note</gi> or <gi scheme="MEI">rest</gi> elements is only acceptable in Common Music Notation, not mensural).</p>
```
```suggestion
                  <p>Dots in mensural notation are not encoded as children of notes or rests, but rather as a sibling of these. They are also not encoded as attributes (the use of the <att>dot</att> attribute in a <gi scheme="MEI">note</gi> or <gi scheme="MEI">rest</gi> element is only acceptable in Common Music Notation, not mensural).</p>
```

martha-thomae committed:
Update source/docs/05-mensural.xml

Co-authored-by: Stefan Münnich <stefan.muennich@unibas.ch>

musicEnfanthen commented:
@ahankinson There still seem to be changes requested although every conversation was resolved. Could you have a look please?

ahankinson commented:
I don't see any pending conversations...

bwbohl commented on source/docs/05-mensural.xml:
```diff
@@ -382,7 +382,7 @@
                      </specList>
                   </p>
-                  <p>PLACE-HOLDER FOR EXAMPLE OF A NOTE WITH TWO STEMS: IMAGE &amp; ITS ENCODING</p>
+                  <p>[include example (image and code) of a note with two stems]</p>
```
Is there already such an example? Otherwise we probably should mark this as XML-comment and add `<!--- TODO -->`before it?

musicEnfanthen commented:
> I don't see any pending conversations...

@ahankinson Yes, me neither. But at the end, GH still throws that there are changes requested (don't know why, maybe it needs another review or your active confirmation that requested changes are adressed?).

ahankinson commented:
Hopefully that fixes it?

musicEnfanthen commented:
Yes, perfect thanks

martha-thomae commented:
Thank you all! 😊
